manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: AutoDesk.DWGTrueView
+PackageVersion: 4.0.19.0
+PackageLocale: en-US
+Publisher: Autodesk, Inc.
+PackageName: Autodesk DWG TrueView
+License: Free

```

```suggestion
License: Proprietary
```
Autodesk has a EULA which is part of the install process, this is not freeware